### ISSUE TITLE
Support for Spawn multiprocessing

### ISSUE THREAD

**Issue by benblack769** [Farama-Foundation/SuperSuit#87]
On MacOS, there are issues with fork multiprocessing as discussed in #75 

Support for spawn multiprocessing is needed to deal with those issues. 

Right now, the workaround solution is to set the process start method to fork manually:

```
import multiprocessing
multiprocessing.set_start_method("fork")
```

And also export this environment variable

```
KMP_DUPLICATE_LIB_OK=TRUE
```

**Comment by benblack769** [comment]
Also `forkserver` support

**Comment by benblack769** [comment]
Resolved by #118
